Cargo.toml

```diff
@@ -34,6 +34,11 @@ openssl-probe = "0.1"
 [target.'cfg(not(any(target_os = "windows", target_os = "macos", target_os = "ios")))'.build_dependencies]
 openssl-src = { version = "300.0.3", optional = true }
 
+[dependencies]
+pem-rfc7468 = { version = "0.2.4", features = ["std"] }
+
 [dev-dependencies]
+pem = "1.0"
+rsa = { version = "0.5.0", features = ["alloc", "pem", "std"] }
```

Comment:
Used to convert RSA keys from `test-cert-gen` to PKCS#8 in tests for `from_pkcs8`.

Reply:
Can you just use a test key that's already in the right format?

Reply:
Rewrote without new dependencies.